samples/zigbee/light_switch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
```

Comment:
`cmake_minimum_required(VERSION 3.13)`, same as was requested in other Zigbee samples.